src/Symfony/Bundle/SecurityBundle/Security.php

```diff
@@ -36,7 +37,7 @@
  *
  * @final
  */
-class Security extends LegacySecurity
+class Security extends LegacySecurity implements UserAuthorizationCheckerInterface
```

Comment:
This class should definitely implement ServiceSubscriberInterface also, to describe what it consumes from the injected container. Can you update it to do so?

Reply:
Added.